web/src/integration_url_modal.ts

```diff
@@ -302,21 +292,21 @@ export function show_generate_integration_url_modal(api_key: string): void {
             get_options: get_options_for_integration_input_dropdown_widget,
             item_click_callback: integration_item_click_callback,
             $events_container: $("#generate-integration-url-modal"),
-            default_id: default_integration_option.unique_id,
```

Comment:
We can still pass `default_id` as `""` and then we would need to call `render()` below after `setup()`.